chirp/drivers/uv5r.py

```diff
@@ -1104,14 +1129,17 @@ def _my_upper_band(self):
         return band_tag
 
     def _get_settings(self):
+        _mem = self._memobj
         _ani = self._memobj.ani
         _fm_presets = self._memobj.fm_presets
         _settings = self._memobj.settings
         _squelch = self._memobj.squelch_new
         _vfoa = self._memobj.vfoa
         _vfob = self._memobj.vfob
         _wmchannel = self._memobj.wmchannel
-        HN5RV = "HN5RV" in str(self._memobj.firmware_msg.line1)
+        _has_fmradio = (str(_mem.bcstfmlo) == "   FM   65-75M" and
+                        str(_mem.bcstfmhi) == "   FM  76-108M")
```

Comment:
Hmm, these strings are in the memory image and not changeable by the user?

Reply:
They not changeable by the typical CHIRP user because they are not exposed to the user. But they are not immutable like the firmware version is in Aux memory block, if that's what you mean. They could be changed by someone that really wanted to change them.

It is for this reason that I block the upload of this region of memory if either or both the image or radio does not have these strings so they can't get overwritten by normal CHIRP use.